app/javascript/components/admin/users/DeleteUser.jsx

```diff
@@ -1,30 +1,66 @@
 import React from 'react';
 import Card from 'react-bootstrap/Card';
+import PropTypes from 'prop-types';
 import {
-  Row, Col, Tab,
+  Row, Col, Tab, Stack, Container,
 } from 'react-bootstrap';
-import { useParams } from 'react-router-dom';
+import { useParams, useNavigate } from 'react-router-dom';
+import { ArrowCircleLeftIcon } from '@heroicons/react/outline';
 import AdminNavSideBar from '../shared/AdminNavSideBar';
+import DeleteAccount from './DeleteAccount';
+import useUser from '../../../hooks/queries/users/useUser';
+import Spinner from '../../shared/stylings/Spinner';
 
 export default function DeleteUser() {
   const { userId } = useParams();
+  const { isLoading, data: user } = useUser(userId);
+  const navigate = useNavigate();
+
+  if (isLoading) return <Spinner />;
+
   return (
     <div id="admin-panel" className="wide-background">
       <h2 className="my-5"> Administrator Panel </h2>
       <Card className="border-0 shadow-sm">
```

Comment:
Is having the `admin-table` `AdminNavSideBar` inside of a container inside of the `admin-panel` `Tab.container` is essential?
If it is, then it should be essential for all of the other admin components IMO.

After choosing and merging #3591, it's important to ensure the consistency and avoid making changes to the admin panel skeleton without generalizing those rectifications into all of the other admin panel components.

The skeleton that I found and used is:
```
   <div id="admin-panel" className="wide-background">
      <h2 className="my-5"> Administrator Panel </h2>
      <Card className="border-0 shadow-sm">
        <Tab.Container activeKey="active-key">
          <Row>
            <Col sm={3}>
              <div id="admin-sidebar">
                <AdminNavSideBar />
              </div>
            </Col>
            <Col sm={9}>
              <Tab.Content className="p-3 ps-0">
                <h1>Content</h1>
              </Tab.Content>
            </Col>
          </Row>
        </Tab.Container>
      </Card>
    </div>
```
If the skeleton is valid then we should be only able to edit what is inside of the `Tab.Content` otherwise there should be a problem with our skeleton that needs to be addressed in its separate PR.

Reply:
Deleted this file so no need to worry about it anymore :sweat_smile: